src/components/ids-swipe-action/ids-swipe-action.js

```diff
@@ -63,7 +64,24 @@ class IdsSwipeAction extends mix(IdsElement).with(IdsEventsMixin, IdsThemeMixin)
     }
 
     if (this.leftButton && this.swipeType === 'reveal') {
-      this.container.scrollLeft = 85;
+      this.container.style.visibility = 'hidden';
+
+      const self = this;
+      const timeout = renderLoop.register(new IdsRenderLoopItem({
```

Comment:
The use of the Renderloop can be good for the callback below this one, since it needs staggering by 200ms.  For the ones that should kick off during the next paint cycle, Renderloop usage is overkill.  This can be replaced with a call to `requestAnimationFrame` and save some cycles:

```suggestion
      requestAnimationFrame(() => {
        this.container.scrollLeft = 85;
      });
```